packages/terra-date-picker/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Unreleased
 ### Changed
 * Resolved lint warnings for multiple empty lines
 * Regenerate wdio screenshots
+* Fixed some wdio test logic
```

Comment:
This should be under `### Fixed`. Also can you be a bit more descriptive?

Reply:
Fixed Here https://github.com/cerner/terra-framework/pull/962/commits/78ec9821e8de85d34678c4514e45c0b21db37209